src/sentry/api/serializers/rest_framework/rule.py

```diff
@@ -30,8 +30,25 @@ def from_native(self, data):
             msg = "Invalid node. Could not find '%s'"
             raise ValidationError(msg % data['id'])
 
-        if not cls(self.context['project'], data).validate_form():
-            raise ValidationError('Node did not pass validation')
+        node = cls(self.context['project'], data)
+
+        if not node.form_cls:
+            return data
+
+        form = node.get_form_instance()
+
+        if not form.is_valid():
+            # XXX(epurkhiser): Very hacky, but we really just want validation
+            # errors that are more specific, not just 'this wasn't filled out',
+            # give a more generic error for those.
+            first_error = form.errors.values()[0][0]
```

Comment:
Better to use `next(six.itervalues(form.errors))[0]` for python3 compat.

Reply:
c4f8d7e. thanks agreed.